Modules/Filtering/ImageFrequency/include/itkFrequencyImageRegionConstIteratorWithIndex.h

```diff
@@ -174,8 +174,10 @@ class ITK_TEMPLATE_EXPORT FrequencyImageRegionConstIteratorWithIndex : public Im
   void
   Init()
   {
-    this->m_FrequencyOrigin = this->m_Image->GetOrigin();
-    this->m_FrequencySpacing = this->m_Image->GetSpacing();
+    for (unsigned int dim = 0; dim < ImageType::ImageDimension; ++dim) {
```

Comment:
Was this set of changes necessary?

Reply:
Yeah, different types

```cpp
Modules/Filtering/ImageFrequency/include/itkFrequencyImageRegionConstIteratorWithIndex.h:177:29:
 error: no match for ‘operator=’ (operand types are ‘itk::FrequencyImageRegionConstIteratorWithIndex<itk::Image<double, 3> >::FrequencyType’ {aka ‘itk::Vector<double, 3>’} and
  ‘const itk::ImageBase<3>::PointType’ {aka ‘const itk::Point<double, 3>’})
  177 |     this->m_FrequencyOrigin = this->m_Image->GetOrigin();
  ```

Reply:
Is there a specific reason why `m_FrequencyOrigin` and `m_Image->GetOrigin()` have different types? I mean, why not declare  `m_FrequencyOrigin` as an `itk::Point` as well?

Reply:
I don't exactly remember the reasons, but I think Vector has more math operations operators than Point.

All the other frequency iterators are also itk::Vector. If you feel like changing it, open an issue and let's discuss it, but it is out of the scope of this fix.